src/client/datascience/telemetry/telemetry.ts

```diff
@@ -55,36 +44,6 @@ const trackedInfo = new Map<string, [ResourceSpecificTelemetryProperties, Contex
 const currentOSType = getOSType();
 const pythonEnvironmentsByHash = new Map<string, PythonEnvironment>();
 
-export function getErrorClassification(error: Error) {
```

Comment:
Removed this gian if block, now all errors inherit BaseError & the informatino is in the error object